oonib/testhelpers/http_helpers.py

```diff
@@ -168,7 +190,367 @@ def all(self, length, keyword):
             length = 100000
         self.write(self.genRandomPage(length, keyword))
 
+
+META_CHARSET_REGEXP = re.compile('<meta(?!\s*(?:name|value)\s*=)[^>]*?charset\s*=[\s"\']*([^\s"\'/>]*)')
+
+def representBody(body):
+    # XXX perhaps add support for decoding gzip in the future.
```

Comment:
So, it would probably make sense to move this piece of code in `txextra`. I think it would be okay to do this refactoring later and for now only add a ticket for it.